### PR TITLE
Update Maven wrapper plugin settings to simplify PR #1769

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Updates version of Maven wrapper from older Takari one to new Apache one; also updates Maven version to d/l from 3.6.3 to 3.8.5.
Not much functional difference but this is the new version recommended for use.

Part of changes from #1769.

**Which issue(s) this PR fixes**:

No separate issue filed

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -492,6 +492,10 @@
           <artifactId>maven-failsafe-plugin</artifactId>
           <version>3.0.0-M3</version>
         </plugin>
+        <plugin>
```
Comment (tatu-at-datastax) [inline pom.xml:495]: Not sure why, but without adding definition here, use of `mvn wrapper:wrapper` fails.

**Review by jeffreyscarpenter — APPROVED** [review]

**Comment by tatu-at-datastax** [comment]
@ivansenic here's my attempt to add one part of changes #1769 as per my comments. Was not 100% sure where `.mvn/wrapper/MavenWrapperDownloader.java` comes from, may add it if I find a more up-to-date one.

**Comment by ivansenic** [comment]
> @ivansenic here's my attempt to add one part of changes #1769 as per my comments. Was not 100% sure where `.mvn/wrapper/MavenWrapperDownloader.java` comes from, may add it if I find a more up-to-date one.

Could be Quarkus generated stuff.. Not sure.. But if we don't needed than it's fine..

**Comment by tatu-at-datastax** [comment]
@ivansenic yeah that's fine. I did use version you had since it's bit newer. Most tutorials simply mention it should exists but not where it comes from. :)

**Pull request merged**